### ISSUE TITLE
🔗 🐞 Bug Report: Lightbox images on Gallery Widged oversize image (Merged with #21413) - [ED-9856]

### ISSUE THREAD

**Issue by contrastxxx** [elementor/elementor#21384]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The images should shrink to the size of the window, but it doesn't happen

Elementor and Pro 3.11.1 

### Steps to reproduce

open photos in gallery 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.5.17
	PHP Version: 8.0.27
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://cassoviacode.com
	Home URL: https://cassoviacode.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Europe/Bratislava
	Admin Email: mikulas.zacok@cassoviacode.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Complianz | GDPR/CCPA Cookie Consent
		Version: 6.4.1
		Author: Really Simple Plugins

	Elementor
		Version: 3.11.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.11.1
		Author: Elementor.com

	EWWW Image Optimizer
		Version: 6.9.3
		Author: Exactly WWW

	Happy Elementor Addons
		Version: 3.8.2
		Author: weDevs

	Pods - Custom Content Types and Fields
		Version: 2.9.11.1
		Author: Pods Framework Team

	Rank Math SEO
		Version: 1.0.108
		Author: Rank Math

	Simple Local Avatars
		Version: 2.7.3
		Author: 10up

	Wordfence Security
		Version: 7.9.0
		Author: Wordfence


== Elements Usage ==
	
	container : 1
		container : 2
		heading : 1
		image : 8
	footer : 1
		container : 6
		heading : 3
		icon-list : 3
		image : 2
		social-icons : 1
		text-editor : 2
		theme-site-logo : 1
	header : 1
		container : 2
		icon : 1
		icon-list : 1
		nav-menu : 1
		social-icons : 1
		theme-site-logo : 1
	popup : 1
		container : 1
		icon-list : 1
		nav-menu : 1
		social-icons : 1
		theme-site-logo : 1
	single : 1
		button : 1
		heading : 3
		image : 4
	wp-post : 4
		button : 2
		container : 4
		heading : 16
		icon-list : 2
		image : 10
		image-carousel : 2
		media-carousel : 2
		text-editor : 32
	wp-page : 16
		button : 11
		container : 153
		counter : 9
		form : 2
		gallery : 2
		google_maps : 1
		ha-horizontal-timeline : 1
		heading : 110
		icon : 23
		icon-box : 33
		icon-list : 3
		image : 86
		image-box : 3
		image-carousel : 4
		image-gallery : 1
		lottie : 4
		posts : 6
		social-icons : 1
		spacer : 3
		text-editor : 61
	single-post : 1
		button : 1
		divider : 2
		heading : 2
		icon-box : 1
		posts : 1
		theme-post-content : 1
		theme-post-featured-image : 1
	single-page : 1
		heading : 1
		post-info : 1
		theme-post-content : 1
	error-404 : 1
		button : 1
		heading : 1
		image : 4


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap
	load_fa4_shim: yes


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Default to New Theme Builder: Active
	Hello Theme Header & Footer: Inactive
	Editor V2: Inactive by default
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Save as Default: Active
	Page Transitions: Inactive
	Notes: Active
	Loop: Active
	Form Submissions: Active
	Scroll Snap: Active
	Mega Menu: Inactive by default


== Log ==
	
Log: showing 20 of 382023-02-15 10:09:20 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]
2023-02-15 10:09:20 [info] elementor::elementor_updater Started 
2023-02-15 10:09:20 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-15 10:09:21 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-15 10:09:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]
2023-02-15 15:11:40 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-15 15:11:40 [info] elementor-pro::elementor_pro_updater Started 
2023-02-15 15:11:40 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-15 15:11:40 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-15 15:11:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-16 12:17:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-16 12:17:16 [info] elementor-pro::elementor_pro_updater Started 
2023-02-16 12:17:16 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-16 12:17:16 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-16 12:17:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-16 12:17:25 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-16 12:17:26 [info] elementor::elementor_updater Started 
2023-02-16 12:17:26 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-16 12:17:26 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-16 12:17:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]

PHP: showing 3 of 3PHP: 2023-02-13 13:17:51 [warning X 13][/data/d/a/da512a57-c343-4a79-8ed9-43539d72c510/cassoviacode.com/web/wp-content/plugins/elementor/app/modules/import-export/module.php::628] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-13 13:17:53 [warning X 13][/data/d/a/da512a57-c343-4a79-8ed9-43539d72c510/cassoviacode.com/web/wp-content/plugins/elementor/core/common/modules/ajax/module.php::172] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-16 15:43:08 [warning X 1][/data/d/a/da512a57-c343-4a79-8ed9-43539d72c510/cassoviacode.com/web/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2023-02-15 10:46:20 [error X 2][https://cassoviacode.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.0:3:670939] this.model.isValidChild is not a function 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Happy Elementor Addons: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Happy Elementor Addons: Incompatible
				

````

</details>

**Comment by Ashic90** [comment]
because of upgrade swiper library ...

**Comment by contrastxxx** [comment]
> because of upgrade swiper library ...

Thank you, it works

**Comment by nicholaszein** [comment]
Hello @contrastxxx! 

Thank you for submitting your issue! 🙏 

🏷️ This issue has been labeled and will be forwarded to the team for validation and possible solution. Please keep in mind that we can't provide any ETA on when this issue could be fixed if it is validated. We ask for patience.

Best regards

**Comment by nicholaszein** [comment]
Hello @contrastxxx! 

Thank you for submitting your issue! 🙏 

We are aware of a conflict happening with the new "Upgrade Swiper Library" experiment, and a fix is underway. For now, we are closing and merging this issue with #21413 which is on the same subject.

Feel free to continue the conversation there.

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.11.3**! 🥳 

✅ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
